### PR TITLE
Allow to inject customized projects when the build is used for other project types than C#/F#

### PR DESCRIPTION
**Pull request by bmoyet**
In our case we need to inject Rust projects graph. It seemed to me more appropriate to not handle this new language in this library which is focused on .NET. This way it allows the build users to inject their own projects with the correct dependencies (Rust, Node...)

### CODE REVIEW

**Comment by sergii-s** [comment]
Good idea, I will check implementation !

File: fb2/fb2.fsproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <AssemblyName>IncrementalBuild</AssemblyName>
+    <AssemblyName>AntVoiceBuild</AssemblyName>
```
Comment (sergii-s) [inline fb2/fb2.fsproj:5]: heh

Comment (AntVoice-Tech) [reply fb2/fb2.fsproj:5]: Yep, you can cancel this PR for your github repo I think, I customized it a bit to be able to deploy our own version :)

Comment (bmoyet) [reply fb2/fb2.fsproj:5]: (It's me writing)

**Pull request closed without merging**